src/com/owncloud/android/ui/fragment/FileDetailFragment.java

```diff
@@ -217,6 +217,24 @@ public void onPrepareOptionsMenu(Menu menu) {
             item.setVisible(false);
             item.setEnabled(false);
         }
+
+        item = menu.findItem(R.id.action_switch_view);
```

Comment:
This shouldn't be this way. In tablets in landscape mode, when a file is shown in the right pane, the options for the current folder still make sense. Please, undo this.